server/controllers/stock/import.js

```diff
@@ -94,6 +94,28 @@ function importStock(req, res, next) {
     .done();
 }
 
+/**
+ * @function formatInt
+ * @param {string} quantity
+ * @description
+ * remove number formatting commas and convert into integer
+ * the decimal separator accepted is . not ,
+ */
+function formatInt(quantity) {
+  return typeof (quantity) === 'string' ? parseInt(quantity.replace(/,/g, ''), 10) : quantity;
+}
+
+/**
+ * @function formatDecimal
+ * @param {string} quantity
+ * @description
+ * remove number formatting commas and convert into a decimal (float)
+ * the decimal separator accepted is . not ,
+ */
+function formatDecimal(quantity) {
+  return typeof (quantity) === 'string' ? parseFloat(quantity.replace(/,/g, '')) : quantity;
```

Comment:
What happens if someone submits "10,00"?  Will this translate it into "1000" or "10.00"?  It looks like the former when it should be the second one.

Reply:
1000

Reply:
Maybe we must find a best way to handle that cause the user can set in the quantity column : "10,000.00" and "10,05" for example, the current implementation accepted the US formatting "." as a separator of decimal

Reply:
Could we just throw an error if it is not in the USD formatting?  In other words, if it isn't a JS Number type, we just break?  That way, we don't accidentally make a mistake in our parsing.